.github/workflows/release.yml

```diff
@@ -278,19 +278,55 @@ jobs:
           packages_dir: wheels/
           repository_url: https://test.pypi.org/legacy/
 
-  build-docs:
+  create-binder-env:
     needs: [deploy]
     runs-on: ubuntu-latest
+    steps:
+      - name: Get scikit-decide release version and define new binder env tag
+        run: |
+          TAG_NAME=${GITHUB_REF/refs\/tags\//}  # stripping "refs/tags/"
+          SKDECIDE_VERSION=${TAG_NAME/v/}  # stripping "v"
+          BINDER_TAG=binder_${TAG_NAME}
+          echo "BINDER_TAG=${BINDER_TAG}" >> $GITHUB_ENV
+          echo "SKDECIDE_VERSION=${SKDECIDE_VERSION}" >> $GITHUB_ENV
+
+      # Checks-out binder branch
+      - name: checkout binder branch
+        uses: actions/checkout@v2
+        with:
+          ref: binder
+
+      - name: Update scikit-decide dependency and tag the environement
+        run: |
+          sed -e "s/scikit-decide\[all\]==.*/scikit-decide\[all\]==${SKDECIDE_VERSION}/" environment.yml > environment.yml.new
+          mv environment.yml.new environment.yml
+          git config user.name "Binder Bot"
+          git config user.email "binder@bot.com"
```

Comment:
Which value is set if user.email empty?  A github one, or the mail from the person who pushed the tag?

Reply:
From my test it creates it automatically and it gives something like "runner@fv-az36-926.1s0p3okewbbu3agytf0rxb3dlg.gx.internal.cloudapp.net"